src/ctia/properties.clj

```diff
@@ -20,13 +22,40 @@
 (defonce properties
   (atom {}))
 
+(defn default-store-properties [store]
+  {(str "ctia.store." store) s/Str})
+
+(defn sql-store-impl-properties [store]
+  {(str "ctia.store.sql." store ".classname") s/Str
+   (str "ctia.store.sql." store ".subprotocol") s/Str
+   (str "ctia.store.sql." store ".subname") s/Str
+   (str "ctia.store.sql." store ".delimiters") s/Str})
+
+(defn es-store-impl-properties [store]
+  {(str "ctia.store.es." store ".uri") s/Str
```

Comment:
This is probably something we could standardize around, Yann.   Probably shouldn't block this PR, but we could address it in future refactoring and come up with a standard way of doing it moving forward.